.env

```diff
@@ -0,0 +1,27 @@
+# URL of the Ethereum node
+URL=http://localhost:8545
+# Default to 24 hours
+TIMEOUT=86400000
+RETRY_INTERVAL=10000
+POLLING_INTERVAL=30000
+CONFIRMATIONS=1
+# Default to 10 minutes
+CONFIRMATION_TIMEOUT=600000
+GAS_LIMIT_MULTIPLIER=160
+GAS_PRICE_MULTIPLIER=160
+BALANCE_THRESHOLD=0.05
+# Gas price predictor strategy. One of < eth-provider | fast | fastest | safeLow | average >
+GAS_PRICE_PROVIDER=eth-provider
+
+# ETH Gas Station API parameters
+GAS_STATION_API_CHAIN_ID=1
+GAS_STATION_API_URL=https://ethgasstation.info/json/ethgasAPI.json
+GAS_STATION_API_KEY=
+GAS_STATION_API_REQUEST_TIMEOUT_MS=10000
+
+# When loading wallet from MNEMONIC
+MNEMONIC=
```

Comment:
Not a good idea to have the option to set the MNEMONIC here, some users might set this and it's definitely very unsafe.